spinnaker/resource_pipeline.go

```diff
@@ -63,12 +59,20 @@ func resourcePipelineCreate(data *schema.ResourceData, meta interface{}) error {
 
 	tmp["application"] = applicationName
 	tmp["name"] = pipelineName
+	delete(tmp, "id")
 
 	if err := api.CreatePipeline(client, tmp); err != nil {
 		return err
 	}
 
-	//data.SetId(fmt.Sprintf("%s/%s", applicationName, pipelineName))
+	// We could set an id here manually, this is done by spinnaker automatically though.
+	// Since we read the generated pipeline and set the Id at the end of this function anyway, we don't need to do this.
+	// uid := uuid.NewV4().String()
+	// err := data.Set("pipeline_id", uid)
+	// if err != nil {
+	// 	return fmt.Errorf("Could not set pipeline_id for pipeline %s", pipelineName)
+	// }
+	// data.SetId(uid)
```

Comment:
Nice catch! Let's go ahead and remove this commented out code! We can add it back later if we need to!

Reply:
updated the PR :)